froide/foirequest/tests/test_helper.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+
+from froide.foirequest.models import FoiAttachment
+from froide.helper.storage import get_numbered_filename
+
+
+class HelperTest(TestCase):
```

Comment:
This should have been a unit test not needing the django stuff, but I did not find any so far. So I wasn't sure whether and how to set it up... What do you propose?